sylius/return-plugin/0.1/manifest.json

```
@@ -0,0 +1,9 @@
+{
+    "aliases": ["return"],
```

A little bit more bulletproof :D 

```suggestion
    "aliases": ["return", "returns"],
```